homeassistant/components/fronius/sensor.py

```diff
@@ -591,13 +591,15 @@ async def async_setup_entry(
     SensorEntityDescription(
         key="capacity_maximum",
         name="Capacity maximum",
-        native_unit_of_measurement=ENERGY_WATT_HOUR,
+        native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
```

Comment:
I think Capacity designed and maximum should have the device class set to ENERGY.